projects/igniteui-angular/src/lib/navigation-drawer/themes/_base.scss

```diff
@@ -0,0 +1,265 @@
+@use 'igniteui-theming/sass/typography' as *;
+@use 'igniteui-theming/sass/bem' as *;
+@use 'igniteui-theming/sass/themes' as *;
+@use 'igniteui-theming/sass/animations' as *;
+@use 'igniteui-theming/sass/color/functions' as *;
+@use 'styles/themes/standalone' as *;
+@use 'light/tokens' as *;
+
+$_theme: $material;
+
+@include layer(base) {
+    %item {
+        position: relative;
+        display: flex;
+        align-items: center;
+        flex-flow: row nowrap;
+        color: var-get($_theme, 'item-text-color');
+        max-height: var(--item-min-height);
+        gap: var(--item-gap);
+        padding: var(--item-padding);
+        cursor: pointer;
+        user-select: none;
+        outline: transparent;
+        white-space: nowrap;
+        border-radius: var-get($_theme, 'item-border-radius');
+        text-decoration: none;
+        border: none;
+        justify-content: flex-start;
+
+        igx-icon {
+            --component-size: 3;
+
+            color: var-get($_theme, 'item-icon-color');
+        }
+
+        &[igxButton] {
+            background: transparent;
+            border: none;
+            box-shadow: none;
+
+            igx-icon {
+                width: var(--igx-icon-size, rem(24px));
+                height: var(--igx-icon-size, rem(24px));
+                font-size: var(--igx-icon-size, rem(24px));
+                margin: 0;
+            }
+
+            &:hover,
+            &:focus {
+                box-shadow: none;
+                border: none;
+            }
+        }
+
+        &%igx-button--fab {
+            min-height: auto;
+        }
+
+        &%igx-icon-button-display {
+            height: auto;
+            transition: none;
+        }
```

Comment:
```scss
        &%igx-button--fab {
            min-height: auto;
        }

        &%igx-icon-button-display {
            height: auto;
            transition: none;
        }
```

You shouldn't use the placeholder selectors when referencing other components. I feel this wasn't tested and thus it wasn't found out it simply doesn't work.